README.md

```diff
@@ -2,130 +2,113 @@
 
 # configen
 
-A command line tool to auto-generate configuration file code, for use in Xcode projects. To read about the motivation behind this project and the scenarios in which it might be used see our [blog post](http://www.theappbusiness.com/blog/a-robust-multi-environment-build-setup). 
+A command line tool to auto-generate configuration file code, for use in Xcode projects.
 
-The `configen` tool is used to auto-generate configuration code from a property list. It is intended to
-create the kind of configuration needed for external URLs or API keys used by your app. Currently supports both Swift and Objective-C code generation.
+The `configen` tool is used to auto-generate configuration code from a Swift file. It is intended to
+create the kind of configuration needed for external URLs or API keys used by your app. It only supports Swift code.
 
 # Installation
 
-To add the `configen` tool to your project you must first aquire the `configen` excecutable binary. The simplest way to do this is to download the executable binary from the latest release.
+To add the `configen` tool to your project, you must first aquire the `configen` excecutable binary and the necessary frameworks. The simplest way to do this is to download the executable binary and frameworks from the latest release.
 
-Alternatively you can download or clone this repository. Once you have done so, open and build the `configen.xcodeproj` project in Xcode, right click on the `configen` product and select ‘Show in Finder’.
+Alternatively, you can download or clone this repository. Once you have done so, open and build the `configen.xcodeproj` project in Xcode, right click on the `configen` product and select ‘Show in Finder’.
```

Comment:
People will need to resolve the package dependencies and regenerate the package (I wanted to try out the "insert a suggestion" feature) 
```suggestion
Alternatively, you can download or clone this repository. Once you have done so, open a terminal window, `cd` to the repository's folder, and run `swift package update` to download the package dependencies, then `swift package generate-xcodeproj` to regenerate the Xcode project. Open and build the `configen.xcodeproj` project in Xcode, right click on the `configen` product and select ‘Show in Finder’.
```

Reply:
Yup, good catch!